python/grass/temporal/c_libraries_interface.py

```diff
@@ -1768,9 +1743,7 @@ def available_mapsets(self):
         """
         self.check_server()
         self.client_conn.send(
-            [
-                RPCDefs.AVAILABLE_MAPSETS,
-            ]
+            [RPCDefs.AVAILABLE_MAPSETS,]
```

Comment:
Black wants to revert this change.

Reply:
Reverting these will also solve Flake8's failure.

Reply:
This is a problem, as this change **is** made by running black on the file... I'll try to ignore black and edit file manually.

Reply:
Testing this locally with Black 20.8b1, gives the same result as on CI's Black 21.5b1 (#1605).

Reply:
The latest version of black shipped for my distribution (Ubuntu 20.10; black, version 19.10b0) considers my version to be the correct one. This is a more serious issue as we expect contributors to run black on source before commit, but the outcome will depend on the version of black in use. And this is not a case of me running some ancient distribution. Latest Ubuntu LTS ships black 19.3b0 and thus will be around for a long time. @wenzeslaus comments?
I could edit the file manually, but AFAIU avoiding manual formatting of code was the main reason of black'ifying in the first place.

Reply:
By default, Black is _using trailing commas as a reason to split lines_ (disabled by `--skip-magic-trailing-comma` which we are currently not using). There was some change between v19 and v20 which causes this loop of satisfying one but not the other, but I didn't investigate further. Black is in beta, so this is expected, but we commit to using Black because it is better than the alternative. Did you try to remove the trailing comma?

What you can do with older systems is to install Black with `pip` into `~/.local` (my current choice) or get a Docker image and set a alias (my previous choice). What we could do in the CI is to apply Black and commit to the PR's branch, but that's much more complicated and potentially confusing for beginners.

Reply:
This is a bit of annoyance we have to live with until there is a stable release of Black (which is also widely distributed). The version on CI is the latest and should have precedence. The log files of the CIs are a treasure trove, if you look at *details* on "Python Black Formatting / Check" you will see the diff of Black's suggested changes, the eventual minor deviations of local Black formatting and the CIs may be applied manually.

Reply:
Fixed with black in a virtualenv.